src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -55,6 +68,10 @@ export default {
 			addToBasketExpActive: false,
 			pageLayout: 'control',
 			addBundlesExp: false,
+			meta: {
+				title: '',
+				description: ''
```

Comment:
Setting the title and description as `undefined` instead of empty strings will allow the default title/description to be used when one isn't set (like the world-refugee-day category which doesn't have a description) https://vue-meta.nuxtjs.org/guide/special.html#use-child-property-conditionally

Reply:
Good to know, thank you!